### ISSUE TITLE
schedule() command doesn't defaulty take into account instruction map.

### ISSUE THREAD

**Issue by patdenim** [Qiskit/qiskit#10837]
### Environment

- **Qiskit Terra version**: 0.25.0
- **Python version**: 3.10
- **Operating system**: Win11

### What is happening?

schedule() command fails when the circuit includes 'measure' with the error message

 ```
schedule(circuit, backend)  

 qiskit.pulse.exceptions.PulseError: "Operation 'measure' exists, but is only defined for qubits [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26]."

```

The command works when specifying the default instruction map 

`schedule(qc, backend, backend.defaults().instruction_schedule_map, meas_map=backend.configuration().meas_map )
`

### How can we reproduce the issue?

```
 provider = IBMProvider()
 qc = QuantumCircuit(1)
 qc.h(0)
 qc.measure_all()  # or qc.measure(0)
 # Schedule
 qctr = transpile(qc, backend_nisq, initial_layout=None, optimization_level=0)
 sch = schedule(qctr, backend)

```

### What should happen?

default instruction and default measure maps should be defaultly taken into account without having to explicitly specify it in the schedule() command

**Comment by MozammilQ** [comment]
@patdenim ,
I am unable to reproduce the error as shown by you.
This is what I am getting instead!
Could you please confirm if you are getting the same, I would like to work on this issue :)
I have checked it, the same is the problem with qiskit verison 0.44.2 with qiskit-terra version 0.25.2.
![Screenshot_20231006_211526](https://github.com/Qiskit/qiskit/assets/54737489/c6edc251-3724-499f-b95b-f5419320c71c)

**Comment by MozammilQ** [comment]
@fvarchon , am I allowed to work on this issue?

**Comment by fvarchon** [comment]
@MozammilQ Yes for sure :)

**Comment by MozammilQ** [comment]
@fvarchon , sir, please see if my efforts a good enough :), please let me know if any additional changes are required :)

**Comment by patdenim** [comment]
Thanks @MozammilQ, you've got the point !! fails only when backends are V1.

**Comment by jakelishman** [comment]
@to24toro, @nkanazawa1989: was this fully closed by #10988 or is there more to do?

**Comment by to24toro** [comment]
@jakelishman 
I think we can close this issue.
